android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1784,6 +1789,68 @@ public Profile getCurrentProfile() {
         return Profile.fromWebContents(tab.getWebContents());
     }
 
+    /** Close all tabs whose URL origin matches with a given origin. */
+    public void closeAllTabsByOrigin(@NonNull final String origin) {
+        final Tab activeTab = getActivityTab();
```

Comment:
Doesn't current Activity tab included in `Set<Integer> tabIndexes = getTabIndexesByUrlOrigin(tabModel, origin);`? I think it should return it's index there as well.

Reply:
Yes, you're right! Good catch! I'm going to refactor this part.

Reply:
Changes done in commit [`91727f8` (#24960)](https://github.com/brave/brave-core/pull/24960/commits/91727f880a0647d3271072b698cfd760dddad24d)